### ISSUE TITLE
Needs Preferences GUI for BS-X BIOS

### ISSUE THREAD

**Issue by tempura666** [OpenEmu/OpenEmu#2527]
Some of the SNES BS game title in order to work properly, you need BSX-BIOS.

In snes9X core supports the BSX-BIOS.
However, Preferences -> System Files on OpenEmu, an item to add it does not have GUI.
For this reason, BSX-BIOS must be copied manually to the directory.

Reference) https://github.com/OpenEmu/OpenEmu/wiki/User-guide:-BIOS-files

**Comment by yetisyny** [comment]
It appears to currently be in the GUI of the current version of OpenEmu, being able to install BSX-BIOS for the Satellaview addon for SNES, with the snes9x core. I think this issue has been resolved, since I was able to use the GUI to install all the different BIOSes including both English and Japanese BSX Satellaview BIOSes for SNES, no need to copy anything manually to directories.

This issue should be closed and marked resolved fixed.
